src/libraries/Microsoft.PowerFx.Interpreter/Functions/Mutation/PatchFunction.cs

```diff
@@ -254,11 +260,16 @@ public async Task<FormulaValue> InvokeAsync(FormulaValue[] args, CancellationTok
                 return args[1];
             }
 
-            var argFields = CreateRecordFromArgsDict(args, 1);
+            var newRecord = FormulaValue.NewRecordFromFields(
+                FieldDictToFieldEnumerable(
+                    CreateRecordFromArgsDict(args, 2)));
 
+            var arg0 = (TableValue)args[0];
```

Comment:
>(TableValue [](http://example.com/codeflow?start=23&length=11)

nit - should check arg0 for blank. (like we do on arg1 on 258)

We already check for errors on 251, so that's good. 
(possibly the blank checks could be in CheckArgs)

Reply:
I'll leave each implementation to check arg(0) since it can be blank for some PatchRecordFunction.